Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository = "https://github.com/BurntSushi/ripgrep"
 keywords = ["regex", "grep", "egrep", "search", "pattern"]
 categories = ["command-line-utilities", "text-processing"]
 license = "Unlicense OR MIT"
-exclude = ["HomebrewFormula"]
+exclude = ["HomebrewFormula", "/.github/", "/ci/", "/pkg/", "/tests/data/", "/benchsuite/", "/scripts/"]
```

Comment:
Could you make sure we don't exceed 79 columns here? e.g.,

```toml
exclude = [
    "foo",
    "bar",
    ...
]
```

Also, please drop `/tests/data` from this list. Dropping that means you can't run tests using just the crate download.

Ignoring the rest looks sensible though. Thanks!